frontend/src/components/nutrigenomics/PathwayDetailPanel.tsx

```
@@ -0,0 +1,237 @@
+                <span className="ml-1">
+                  ({detailQuery.data.missing_snps.length} not on array)
+                </span>
+              )}
+            </p>
+          )}
+        </div>
+        <button
+          onClick={onClose}
+          className="rounded-md p-1.5 hover:bg-muted transition-colors"
+          aria-label="Close pathway details"
+        >
```

Biome (2.4.6)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)